apps/rehabstod/src/components/Table/TableFilter.test.tsx

```diff
@@ -17,45 +17,47 @@ const renderComponent = () => {
   )
 }
 
-describe('TableFilter', () => {
```

Comment:
Is this necessary to remove?

Reply:
Neither necessary to remove nor keep, I would say. Doesn't add anything, but doesn't hurt much either. Is there a reason you would want to keep them?

Reply:
Personally I prefer to avoid formatting changes to complete files since it removes the git blame functionality (if one doesn't go in on the git blame history). But it works either way of course